SimGeneral/MixingModule/python/trackingTruthProducer_cfi.py

```diff
@@ -51,9 +51,12 @@
     trackingParticles.simTrackCollection = cms.InputTag('famosSimHits')
     trackingParticles.simVertexCollection = cms.InputTag('famosSimHits')
 
-def _modifyTrackingParticlesForPhase2( object ):
+def _modifyTrackingParticlesForRun3( object ):
     object.simHitCollections.muon.append(cms.InputTag("g4SimHits","MuonGEMHits"))
+
+def _modifyTrackingParticlesForPhase2( object ):
     object.simHitCollections.muon.append(cms.InputTag("g4SimHits","MuonME0Hits"))
 
 from Configuration.StandardSequences.Eras import eras
+eras.run3_GEM.toModify( trackingParticles, func=_modifyTrackingParticlesForRun3 )
```

Comment:
This can be
```python
eras.run3_GEM.toModify(trackingParticles, simHitCollections = dict(
                          muon = trackingParticles.simHitCollections.muon+cmsInputTag("g4SimHits","MuonGEMHits")))
```